UndertaleModLib/Util/FileBinaryWriter.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Buffers.Binary;
+using System.IO;
+using System.Text;
+
+namespace UndertaleModLib.Util
+{
+    public class FileBinaryWriter : BinaryWriter
+    {
+        private readonly Encoding encoding = new UTF8Encoding(false);
+
+        public uint Position
+        {
+            get => (uint)OutStream.Position;
+            set => OutStream.Position = value;
+        }
+
+        public Encoding Encoding { get => encoding; }
+        public Stream BaseStream => OutStream;
```

Comment:
Compiler warning here, that that `FileBinaryWriter.BaseStream` hides `BinaryWriter.BaseStream`.
from what I can see, you probably want to override it?

Reply:
Actually, no, that's just an oversight - it's not even used, so it can be removed